app/Http/Controllers/AdminController.php

```diff
@@ -163,4 +163,12 @@ public function showInbox()
 
         return view('admin.inbox.index')->with('all_messages',$all_messages);
     }
+    public function updateStatus(Request $request, $id)
```

Comment:
to save few lines of code you can actually change `$id` to `Contact $message` and remove the entire `$message = Contact::findOrFail($id);`

in your `web.php` change `{id}` to `{message}`

Reply:
I'll fix it.